libs/pipelines/api_v2_paths.py

```diff
@@ -88,7 +92,12 @@ def bulk_flattened_timeseries_data(self, file_type, state: Optional[str] = None)
     def single_summary(self, region_summary: can_api_v2_definition.RegionSummary, file_type):
         if self.level is AggregationLevel.STATE:
             return self.region_subdir / f"{region_summary.state}.{file_type.suffix}"
-        if self.level in (AggregationLevel.COUNTY, AggregationLevel.CBSA, AggregationLevel.PLACE):
+        if self.level in (
+            AggregationLevel.COUNTY,
+            AggregationLevel.CBSA,
+            AggregationLevel.PLACE,
+            AggregationLevel.COUNTRY,
```

Comment:
I think we can use the country code for the key - I think /country/US.json is clearer and potentially more stable if we end up changing the key

Reply:
Done in ccfc14cb